packages/@okta/courage/src/vendor/lib/backbone.js

```diff
@@ -0,0 +1,1873 @@
+//     Backbone.js 1.2.1
```

Comment:
We didn't have backbone before... do we need it or can we get it from node_modules locally, similar as other vendor files?

Reply:
@mauriciocastillosilva-okta I think we do need to list them as dependencies in package.json file but if we delete it from packages folder we should create aliases for them in the webpack.config file like we have for `handlebars: 'handlebars/dist/handlebars` and point the alias to the dist folder. Let me know if that makes sense and I can update the PR.

Reply:
We do use handlebars and underscore so they need to be listed as dependencies in package.json and we already have mappings for them in the webpack.config so i can delete the files  from packages folder. We don't need backbone and clipboard, so i can delete them as well.

Reply:
Am curious - is there any reason to delete them? I.e. if we're already aliased to the node_modules we're pulling down, it's not that big of a deal that there's a copy in the packages/vendor folder. 

Mostly thinking that if we can simplify the process to upgrade courage (less manual steps), the better.

Reply:
not any reason, it was my understanding that we didn't need duplicates since we could get them from node_modules. 
But you are right, this will make it easier to update courage in the future.

Reply:
Mostly because it was not there in packages folder before. The way I have it now matches the dependency list we have in courage.

Reply:
Great, let's do that then. Btw, is there a doc somewhere outlining the steps we need to update courage @mauriciocastillosilva-okta or @nikhilvenkatraman-okta? If not, we should add this info somewhere.

Reply:
Double checking @nikhilvenkatraman-okta - you didn't manually add this to the vendor folder, did you?

Reply:
@rchild-okta I don't know if there is a doc for this. I'll try and write up a wiki doc.

Reply:
Okay, it seems like we did - as an aside @nikhilvenkatraman-okta, it would've been better to break out the commits with "things that happen automatically by copying courage over" and "everything else". This one's an easy one to miss - nice catch @mauriciocastillosilva-okta.

We don't need to manually copy dependencies back into vendor (in fact, we're moving away from any checked in source files). That's one of the highlights of webpack - it can consume modules directly from the node_modules folder. We've only got aliases for things like handlebars because we had to make an exception - in that case, the main file that's exported by handlebars isn't the one we want to use.

Reply:
@rchild-okta  The files in packages folder were copied over from node_modules after running the update command

Reply:
@rchild-okta In that case I can delete these `4 new` files copied into packages folder and leave the dependencies in package.json ?

Reply:
I don't see backbone.js in the [vendor/lib](https://github.com/okta/courage/tree/master/src/vendor/lib) directory - how did it get there in this PR? Is it part of some build step?

Reply:
Its added to the dependencies list here https://github.com/okta/courage/blob/master/package.json#L100

Reply:
Talked over IM, it's getting copied there as part of the build ([here](https://github.com/okta/courage/blob/master/Gruntfile.js#L84)]. In this case, I think it's fine to just include it - the "update courage" step should just be to copy over everything in the node_modules dir, w/out any extra work to delete things we don't need.

Reply:
@rchild-okta  @mauriciocastillosilva-okta Let me know if you have any other comments, if not i can merge.